.changesets/feat_garypen_next_backpressure.md

```diff
@@ -0,0 +1,9 @@
+### Enabling back-pressure in the request processing pipeline ([PR #6486](https://github.com/apollographql/router/pull/6486))
+
+In Router 1.x, back-pressure was not maintained. Requests would be accepted by the router. This could cause issue for routers which were accepting high levels of traffic.
```

Comment:
Lets go into more detail about the benefits

Reply:
Done, but most of the details will be in the migration guide.